doc/spec.md

```diff
@@ -990,6 +990,12 @@ same name, such as `f(x=1, x=2)`. A call that provides a `**kwargs`
 argument may yet have two values for the same name, such as
 `f(x=1, **dict(x=2))`. This results in a dynamic error.
 
+Function arguments are evaluated in the order they appear in the call.
+
+Unlike Python, Starlark does not allow more than `*args` argument in a
```

Comment:
Should this be "does not allow more than one"?

Reply:
Um, yeah.